Src/Web/Web.Shared.Net.Tests/RequestTrackingTelemetryModuleTest.cs

```diff
@@ -495,10 +521,10 @@ private string GetActivityRootId(string telemetryId)
             return telemetryId.Substring(1, telemetryId.IndexOf('.') - 1);
         }
 
-        private RequestTrackingTelemetryModule RequestTrackingTelemetryModuleFactory(TelemetryConfiguration config = null)
+        private RequestTrackingTelemetryModule RequestTrackingTelemetryModuleFactory(TelemetryConfiguration config = null, CorrelationIdLookupHelper correlationHelper = null)
```

Comment:
No I haven't done that. That's more like a feature request, not critical for build. If you would like to have that exposed, I think we can open a new enhancement issue and track it through that.

What situations would customers want to have this dictionary provided? Some internal testing / PPE thing? I don't think bulk of the customers running on prod care.